### PR TITLE
solves #14

### PR DESCRIPTION
**Pull request by colameo**
@MathieuAA I've removed the blueprint option for vuejs as I assumed we don't need it anymore or am I wrong?

### CODE REVIEW

**Comment by MathieuAA** [comment]
Blueprints should be kept, and yes: blueprint has to be removed.

**Comment by colameo** [comment]
@MathieuAA any progress in this? I know it's summer and hot 😅

**Comment by MathieuAA** [comment]
Oh! Sorry!! I thought it was merged a long time ago. I completely forgot about this one. Don't hesitate to ping as soon as possible (~1 day) if you expect something from me, I have lots of things on my plate right now and I'm not finding the time to do what I always want...

File: JDLApplicationOptions.md [hunk 1]
```diff
@@ -16,22 +16,22 @@
...
+| searchEngine         | STRING   | EnumLiteral           | elasticsearch, couchbase, false                                                        | One of the listed values                                                                                                                        |
 | jhiPrefix            | STRING   | JavaIdentifierLiteral | jhi                                                                                  |                                                                                                                                                 |
-| clientPackageManager | STRING   | EnumLiteral           | npm, yarn                                                                             | One of the listed values or one of the prod database type                                                                                       |
+| clientPackageManager | STRING   | EnumLiteral           | npm, yarn                                                                             | One of the listed values                                                                                                                        |
```
Comment (MathieuAA) [inline JDLApplicationOptions.md:28]: I'll create an issue to remove yarn :)

**Review by MathieuAA — APPROVED** [review]

**Pull request merged**